apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -75,12 +76,14 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
 
         new AgentBuilder.Default(byteBuddy)
             .ignore(nameStartsWith("net.bytebuddy."))
-            .ignore(nameStartsWith("org.slf4j."))
-            .ignore(nameStartsWith("org.apache.logging."))
-            .ignore(nameStartsWith("org.groovy."))
-            .ignore(nameContains("javassist"))
-            .ignore(nameContains(".asm."))
-            .ignore(allSkyWalkingAgentExcludeToolkit())
+                .or(nameStartsWith("org.slf4j."))
```

Comment:
I think this `or` should be inside `ignore` method, is that right? Right now, the codes look like `ignore`.`or`.`or`.

We don't expect for this, right?

Reply:
God，I have made a mistake in using of bracket, sorry, I will check this carefully and send a PR again.

Reply:
I have pushed again.